airflow/opt/providers/etna/etna/auth/etna_cookie_auth.py

```diff
@@ -107,11 +109,59 @@ def sync_project_role(self, project_name: str) -> Role:
         cond = DagModel.tags.any(DagTag.name == project_name)
         dags_query = dags_query.filter(cond)
 
+        project_permissions = [
+            # From standard Viewer
+            (permissions.ACTION_CAN_READ, permissions.RESOURCE_AUDIT_LOG),
```

Comment:
Oh cool!  Dude, thank you for digging into this.  It's the sort of configuration that takes a bit of focus and experimentation to get right.